samples/ScreenLCD_5110/app/application.cpp

```diff
@@ -2,43 +2,38 @@
 #include <SmingCore/SmingCore.h>
 #include <Libraries/Adafruit_PCD8544/Adafruit_PCD8544.h>
 
-// Software SPI (slower updates, more flexible pin options):
-	// pin 7 - Serial clock out (SCLK)
-	// pin 6 - Serial data out (DIN)
-	// pin 5 - Data/Command select (D/C)
-	// pin 4 - LCD chip select (CS)
-	// pin 3 - LCD reset (RST)
-Adafruit_PCD8544 display = Adafruit_PCD8544(14, 13, 12, 5, 4);
-void displayTest()
-{
-	display.begin();
-	  // init done
+// GPIO13/D7 - Serial clock out (SCLK)
+// GPIO12/D6 - Serial data out (DIN)
+// GPIO14/D5 - Data/Command select (D/C)
+// GPIO5/D1 - LCD chip select (CS)
+// GPIO4/D2 - LCD reset (RST)
+Adafruit_PCD8544 display = Adafruit_PCD8544(13, 12, 14, 5, 4);
 
-	  // you can change the contrast around to adapt the display
-	  // for the best viewing!
-	  display.setContrast(10);
+Timer displayTimer;
 
-	  display.display(); // show splashscreen
-	  delay(200);
-	  display.clearDisplay();   // clears the screen and buffer
+void displayTest2()
+{
+	display.clearDisplay();
+	display.setRotation(4);  // rotate 90 degrees counter clockwise, can also use values of 2 and 3 to go further.
+	display.setTextSize(1);
+	display.setTextColor(BLACK);
+	display.setCursor(0, 0);
+	display.println("Sming");
+	display.setTextSize(2);
+	display.println("Example");
+	display.display();
+}
 
-	  display.setRotation(4);  // rotate 90 degrees counter clockwise, can also use values of 2 and 3 to go further.
-	    display.setTextSize(1);
-	    display.setTextColor(BLACK);
-	    display.setCursor(0,0);
-	    display.println("Sming");
-	    display.setTextSize(2);
-	    display.println("Example!");
-	    display.display();
+void displayTest()
+{
+	display.begin();
+	display.setContrast(10);
+	display.display(); // show splashscreen
+	displayTimer.initializeMs(2000, displayTest2).start();
```

Comment:
@zgoda Please, change the code to use `delay(2000)` and remove the use of timers. Since this #1154 PR is merged delays bigger than 1000ms should not cause reboots due to WDT.